data/component_versions.json

```diff
@@ -0,0 +1,20 @@
+{
+    "packages": [
+        {
+            "title": "snowplow_web_latest",
+            "version": "12.4.0"
+        },
+        {
+            "title": "snowplow_ecommerce_latest",
+            "version": "0.1.0"
+        },
+        {
+            "title": "snowplow_mobile_latest",
+            "version": "0.6.2"
+        },
+        {
+            "title": "snowplow_fractribution_latest",
+            "version": "0.1.0"
+        }
+    ]
+}
```

Comment:
So I don't really understand this or the full context for it, but could we not pull the latest (non-prerelease) version via an API call to not have to update this? https://docs.github.com/en/rest/releases/releases?apiVersion=2022-11-28#get-the-latest-release